{"uuid":"66caae5e-8dde-4553-8a72-13efde66e869","historyId":"fda4711033cae3dc26aa695d414df959","fullName":"testClasses.HomeTests.StackDropdown","labels":[{"name":"package","value":"testClasses.HomeTests"},{"name":"testClass","value":"testClasses.HomeTests"},{"name":"testMethod","value":"StackDropdown"},{"name":"parentSuite","value":"DSAlgo Suite"},{"name":"suite","value":"DSAlgo Tests"},{"name":"subSuite","value":"testClasses.HomeTests"},{"name":"host","value":"Dineshs-MacBook-Pro.local"},{"name":"thread","value":"78946@Dineshs-MacBook-Pro.local.TestNG-test-DSAlgo Tests-2(61)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"StackDropdown","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"\nMethod HomeTests.StackDropdown()[pri:0, instance:testClasses.HomeTests@3081f72c] should have thrown an exception of type class java.lang.NumberFormatException","trace":"org.testng.TestException: \nMethod HomeTests.StackDropdown()[pri:0, instance:testClasses.HomeTests@3081f72c] should have thrown an exception of type class java.lang.NumberFormatException\n\tat org.testng.internal.invokers.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:81)\n\tat org.testng.internal.invokers.TestInvoker.considerExceptionsInternal(TestInvoker.java:898)\n\tat org.testng.internal.invokers.TestInvoker.considerExceptions(TestInvoker.java:863)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:742)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)\n\tat org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1575)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1747713514580,"stop":1747713515509}